Project/include/db_query.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+//Use the values from config.php to make a connection to the database. Since this
```

Comment:
This is another place where rather than duplicating the file, I'd stick with the one you've already got.